File: gabaritos/gabarito9.py
numeros = [64, 34, 25, 12, 22, 11, 90]
n = len(numeros)
for i in range(n):
    for j in range(0, n-i-1):
        if numeros[j] > numeros[j+1]:
            numeros[j], numeros[j+1] = numeros[j+1], numeros[j]
print("Lista ordenada:", numeros)

File: csharp/gabaritos/gabarito9.cs
using System;
using System.Collections.Generic;

class Program
{
    static void Main()
    {
        List<int> numeros = new List<int> { 64, 34, 25, 12, 22, 11, 90 };
        int n = numeros.Count;
        for (int i = 0; i < n; i++)
        {
            for (int j = 0; j < n - i - 1; j++)
            {
                if (numeros[j] > numeros[j + 1])
                {
                    int temp = numeros[j];
                    numeros[j] = numeros[j + 1];
                    numeros[j + 1] = temp;
                }
            }
        }
        Console.WriteLine("Lista ordenada: " + string.Join(", ", numeros));
    }
}